test/db/analysis/golang

```diff
@@ -249,7 +249,7 @@ EOF
 EXPECT=<<EOF
 1788
 0x004a699b str.hello__hacktivity
-8555
+9222
```

Comment:
can we have a test to check specifically for the strings that are detected by this new function?

Reply:
i sure can try to find the smallest which doesnt print 9k strings.

Reply:
just `~XXXX` to check if one (or few) specific string(s) are present. no need to print them all.

Reply:
This is the current dev with the new test
![image](https://user-images.githubusercontent.com/561184/183246439-3b193278-cb7b-47d2-b0a7-37ef1a37bf95.png)

Reply:
👍